actionpack/lib/action_dispatch/testing/request_encoder.rb

```diff
@@ -0,0 +1,54 @@
+module ActionDispatch
+  class RequestEncoder # :nodoc:
```

Comment:
`ActionDispatch::TestResponse::RequestEncoder` doesn't exactly seem right either 😄

I'll put it in the `Testing` module which `testing/integration.rb` also uses.